test/entry_set_test.rb

```diff
@@ -66,9 +66,17 @@ def test_entries
   end
 
   def test_find_entry
-    # by default, #find_entry is case-sensitive
-    assert_equal(ZIP_ENTRIES[0], @zipEntrySet.find_entry('name1'))
-    assert_equal(ZIP_ENTRIES[0], @zipEntrySet.find_entry('NaMe1', false))
```

Comment:
But possible situation when rubyzip used in other gems. And you changing behavior for your app but in fast you changed it globally. So need to have both variants. Global and local.